flow/ProtocolVersions.cmake

```diff
@@ -0,0 +1,90 @@
+# Protocol Versions.
+# This version impacts both communications and the deserialization of certain database and IKeyValueStore keys.
+#
+# The convention is that 'x' and 'y' should match the major and minor version of the software, and 'z' should be 0.
+# To make a change without a corresponding increase to the x.y version, increment the 'dev' digit.
+#
+# The last 2 bytes (4 digits) of the protocol version do not affect compatibility. These two bytes are not currently
+# used and should not be changed from 0.
+#                                                          xyzdev
+#                                                          vvvv
+set(FDB_PV_DEFAULT_VERSION                      "0x0FDB00C071200000LL")
+set(FDB_PV_FUTURE_VERSION                       "0x0FDB00C072000000LL")
+set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00B0710A0000LL")
+set(FDB_PV_MIN_INVALID_VERSION                  "0x0FDB00C073000000LL")
```

Comment:
shouldn't this be 0x0FDB00C071400000LL ? (so that 71.3 is compatible but onwards isn't?)

Reply:
Not exactly, it's a little more complicated than that. We don't know if we're going to do 71.3 or apple/7.2 drops first. So realistically we'll need to add another commit and rebuild the downgrade version afterwards.